meta-resin-common/recipes-support/resin-device-pinning/resin-device-pinning/resin-device-pinning.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
Please add license header: see https://github.com/resin-os/meta-resin/pull/1053/files.

Reply:
in the systemd unit?

Reply:
Uh right. Ignore this.